docs/core/install/how-to-detect-installed-versions.md

```diff
@@ -18,13 +18,17 @@ If any command in this article fails, you don't have the runtime or SDK installe
 
 ## Check SDK versions
 
-You can see which versions of the .NET Core SDK are currently installed with a terminal. Open a terminal and run the `dotnet --list-sdks` command.
+You can see which versions of the .NET Core SDK are currently installed with a terminal. Open a terminal and run the following command:
+
+```dotnetcli
+dotnet --list-sdks
+```
+
+You get output similar to the following:
```

Comment:
Editors have indicated that `:` should only be used to to introduce a list, not for a snippet/code.

```suggestion
You get output similar to the following.
```